### PR TITLE
[INLONG-4354][SDK] Fix NPE of the ProxyConfigEntry cluster id

### LINKED ISSUE #4354: [Bug][SDK] ProxyConfigEntry cluster id NPE
### What happened

The properties parsed by the HTTP client are different from those returned by the server.
image
image

### What you expected to happen

The attributes resolved by the HTTP client are the same as those returned by the server.

### How to reproduce

Run org.apache.inlong.sdk.dataproxy.example.TcpClientExample

### Environment

CentOS

### InLong version

1.1.0

### InLong Component

InLong SDK

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by xuesongxs**
Fixes #4354

### CODE REVIEW

**Comment by dockerzhang** [comment]
@luchunliang @baomingyu PTAL

**Review by healchow — APPROVED** [review]

File: inlong-sdk/dataproxy-sdk/src/main/java/org/apache/inlong/sdk/dataproxy/config/ProxyConfigManager.java [hunk 1]
```diff
@@ -588,8 +588,12 @@ private ProxyConfigEntry getLocalProxyListFromFile(String filePath) throws Excep
         Map<String, Integer> streamIdMap = getStreamIdMap(localProxyAddrJson);
         proxyEntry.setGroupIdNumAndStreamIdNumMap(groupIdNum, streamIdMap);
         proxyEntry.setLoad(load);
-        if (localProxyAddrJson.has("cluster_id")) {
-            proxyEntry.setClusterId(localProxyAddrJson.get("cluster_id").getAsString());
+        if (localProxyAddrJson.has("data")) {
```
Comment (baomingyu) [inline inlong-sdk/dataproxy-sdk/src/main/java/org/apache/inlong/sdk/dataproxy/config/ProxyConfigManager.java:591]: The configuration of the cluster_id needs to be added to the local configuration file to deal with this problem. Are you trying to deal with the configuration pulled back from the manager? If this case, you need to modify the requestProxyList function.

**Review by baomingyu — CHANGES_REQUESTED** [review]

File: inlong-sdk/dataproxy-sdk/src/main/java/org/apache/inlong/sdk/dataproxy/config/ProxyConfigManager.java (hunk 1 above)
Comment (baomingyu) [reply inlong-sdk/dataproxy-sdk/src/main/java/org/apache/inlong/sdk/dataproxy/config/ProxyConfigManager.java]: Good job. LGTM. It is suggested to be done in another pr.

**Review by baomingyu — APPROVED** [review]
+1

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**